### ISSUE TITLE
failure loading values with unit specification from configuration file

### ISSUE THREAD

**Issue by AlexeyGusev** [pcb2gcode/pcb2gcodeGUI#15]
File → Load configuration file → Choose your millproject file

An error is thrown during configuration file loading if key=value pair contains unit specification for value, like mm, mm/minute or similar, which is completely legal for pcb2gcode.

Solution: add unit specification processing to file loading routine.

**Comment by eyal0** [comment]
Yes, I know that this needs to be done.  Also, support needs to be added for all the new pcb2gcode flags that were added.  I have not been able to maintain pcb2gcodeGUI but if you want to help, I would like that very much and I will assist you.
